configuration/config-txt/overclocking.md

```diff
@@ -37,21 +37,35 @@ Overclocking and overvoltage will be disabled at runtime when the SoC reaches 85
 
 This table gives the default values for the options on various Raspberry Pi Models, all frequencies are stated in MHz.
 
-| Option       | Pi 0/W | Pi1| Pi2 | Pi3   | Pi3B+ | Pi4  |
+| Option       | Pi 0/W | Pi1 | Pi2 | Pi3   | Pi3A+/Pi3B+ | Pi4  |
 | ---          | ---    | --- | --- | ----  | ----- | ---- | 
 | arm_freq     | 1000   | 700 | 900 | 1200  | 1400  | 1500 |
-| gpu_freq     |400/300 | 250 | 250 |400/300|400/300| 500  |  
-| core_freq    | 400    | 250 | 250 | 400   | 400   | 500  |
-| sdram_fre    | 450    | 400 | 400 | 450   | 500   | 3200 |
+| core_freq    | 400    | 250 | 250 | 400   | 400   | 500/550/432 |
+| h264_freq    | 300    | 250 | 250 | 400   | 400   | 500/550/432 |
+| isp_freq     | 300    | 250 | 250 | 400   | 400   | 500/550/432 |
+| v3d_freq     | 300    | 250 | 250 | 400   | 400   | 500/550/432 |
+| sdram_freq   | 450    | 400 | 400 | 450   | 500   | 3200 |
 | arm_freq_min | 700    | 700 | 600 | 600   | 600   | 600  |
 | core_freq_min| 250    | 250 | 250 | 250   | 250   | 250/275 |
 | gpu_freq_min | 250    | 250 | 250 | 250   | 250   | 500  |
 | h264_freq_min|  250   | 250 | 250 | 250   | 250   | 500  |
 | isp_freq_min |  250   | 250 | 250 | 250   | 250   | 500  |
 | v3d_freq_min |  250   | 250 | 250 | 250   | 250   | 500  |
 | sdram_freq_min |400   | 400 | 400 | 400   | 400   | 400  |
-|overvoltage_min | 0    | 0   |  0  | 0     | 0     | 0    |
-| temp_limit   | 85     | 85  | 85  | 85    | 85    | 85   |
+
+| gpu_freq | Sets `core_freq`, `h264_freq`, `isp_freq`, and `v3d_freq` together; on Pi 3/Pi Zero /Pi Zero W `core_freq` defaults to `400`, while `h264_freq`, `isp_freq`, and `v3d_freq`default to `300`; on Pi 4B this can be set to 500 or 600 only|
```

Comment:
Not sure what the intention here was, but it doesn't render properly as markdown.